tests/Doctrine/Tests/DBAL/Platforms/SQLAnywherePlatformTest.php

```diff
@@ -318,14 +319,14 @@ public function testGeneratesPrimaryKeyDeclarationSQL()
         self::assertEquals(
             'CONSTRAINT pk PRIMARY KEY CLUSTERED (a, b)',
             $this->platform->getPrimaryKeyDeclarationSQL(
-                new Index(null, ['a', 'b'], true, true, ['clustered']),
+                new Index('', ['a', 'b'], true, true, ['clustered']),
```

Comment:
Another way may be:

https://github.com/doctrine/dbal/blob/92920ede68e1d16246712cd41318dd82bb72c015/lib/Doctrine/DBAL/Schema/ForeignKeyConstraint.php#L67-L69